### PR TITLE
[WIP] Create new file for places to eat in Raleigh

### LINKED ISSUE #4: GithubTeacher favorite places to eat
Let's add a document using:
- [x] Create a branch
- [x] Add a new file
- [x] Commit my file
- [x] Open Pull Request
- [ ] Discuss
- [ ] Merge

@carolynshin do you have any specific recommendations?

### PR DESCRIPTION
**Pull request by githubteacher**
:warning: WIP :warning: 
Resolves #4 by adding a file for places to eat in Raleigh.

### CODE REVIEW

File: eats/raleigh-nc.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Raleigh, NC Great Places to Eat
+
+* Aviator's in Fuquay
```
Comment (githubteacher) [inline eats/raleigh-nc.md:3]: Is this spelled correctly @carolynshin

Comment (carolynshin) [inline eats/raleigh-nc.md:3]: @githubteacher, yes Google told me it's correct. Thanks for double checking!

File: eats/raleigh-nc.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+* The Pit for BBQ (North Carolina style)
```
Comment (carolynshin) [inline eats/raleigh-nc.md]: @githubteacher, did you want to add a link to The Pit's Yelp page? http://www.yelp.com/biz/the-pit-raleigh

**Comment by githubteacher** [comment]
:sheep: it!

**Comment by carolynshin** [comment]
:sheep: :x: :100: !

**Pull request merged**